Source/MedicalInfo/MainTabWindow_Medical.cs

```diff
@@ -232,13 +232,40 @@ public override void DoWindowContents(Rect rect)
             var offset = true;
 
             // extra 15f offset for... what? makes labels roughly align.
-            var colWidth = (rect.width - x - 15f) / CapDefs.Count;
-            for (var i = 0; i < CapDefs.Count; i++)
+            var colWidth = (rect.width - x - 15f) / (CapDefs.Count + 2);
+            for (var i = 0; i < 2; i++)
             {
+                var label = offset ? "FluffyMedical.HealthPercentage".Translate() : "PainLevel".Translate();
                 var defLabel = new Rect(x + colWidth * i - colWidth / 2, 10f + ( offset ? 10f : 40f), colWidth * 2, 30f);
                 Widgets.DrawLine(new Vector2(x + colWidth * (i + 1) - colWidth / 2, 40f + (offset ? 5f : 35f)),
                                  new Vector2(x + colWidth * (i + 1) - colWidth / 2, 80f), Color.gray, 1);
-                Widgets.Label(defLabel, CapDefs[i].LabelCap);
+                Widgets.Label(defLabel, label);
+                if (Widgets.ButtonInvisible(defLabel))
+                {
+                    if (OrderBy == Order.Efficiency)
+                    {
+                        Asc = !Asc;
+                    }
+                    else
+                    {
+                        OrderBy = Order.Efficiency;
+                        Asc = true;
```

Comment:
this isn't working. Order.Efficiency is meant to be used together with OrderByCapDef in order to sort by efficiency for a particular capacity def. We'll need to actually implement sorting by health and pain.